tests/cli/test_run.py

```diff
@@ -150,3 +154,31 @@ def test_zt_version(capsys, options):
     assert lines == expected
 
     assert captured.err == ""
+
+
+@pytest.mark.parametrize('option, autohide, no_autohide', [
+        ('--autohide', True, False),
+        ('--no-autohide', False, True),
+        ('--debug', False, False),
+        ]
+)
+def test_autohide_option(capsys, option, autohide, no_autohide):
+    args = parse_args([option])
+    assert args.autohide == autohide
+    assert args.no_autohide == no_autohide
+
+
+@pytest.mark.parametrize('options', [
+        ['--autohide', '--no-autohide'],
+        ['--no-autohide', '--autohide']]
+)
+def test_multiple_autohide_options(capsys, options):
```

Comment:
For test names, it's often useful to start them with the function you're testing (eg. see `test_main_...` or `test_get_login_id`), though I note we've not been completely consistent even in this file.